tensorflow/core/common_runtime/gpu/gpu_bfc_allocator_test.cc

```diff
@@ -568,6 +568,47 @@ class GPUBFCAllocatorPrivateMethodsTest : public ::testing::Test {
     EXPECT_EQ(GPUBFCAllocator::RoundedBytes(1LL << 31),
               force_no_allow_growth_allocator.curr_region_allocation_bytes_);
   }
+
+  void TestRegionDeallocation() {
+    setenv("TF_FORCE_GPU_ALLOW_GROWTH", "unparseable", 1);
```

Comment:
setenv in tests is really frowned upon if tests execute in parallel, is there a different way?

Reply:
Thanks for bringing up this. I double check. I can remove this line.

But please note that (un)setenv has already been used in this class. E.g., see this [line](https://github.com/tensorflow/tensorflow/blob/30f2dff032a27be0d4adbb06979de6f54df1105e/tensorflow/core/common_runtime/gpu/gpu_bfc_allocator_test.cc#L419). This env variable is unset in the `SetUp` of this test so that line 573 is not necessary. The test behavior we need is simply to test with `allow_growth=true` specified by the `GPUOption`. That is why I wanted to unset the env var `"TF_FORCE_GPU_ALLOW_GROWTH"`.